app/models/user.rb

```diff
@@ -1,11 +1,22 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, :confirmable
```

Comment:
- [ ] Good job adding the `confirmable`, but now it does not work as expected because you have not specified the host url for the action mailer to work with.

  In your `config\environments\development.rb` file you need to add this line so that Devise has a host URL to prepend to emails.
 
  ```
  config.action_mailer.default_url_options = { :host => "localhost:3000" }
  ```

  ![image](https://github.com/techmoves/movesBloc/assets/81468675/92c0d382-d41b-40ef-9c91-6b3367e0b392)

Reply:
thanks alot resoved